core/frontend/src/tools/ClipViewTool.ts

```diff
@@ -60,7 +60,7 @@ export interface DrawClipOptions {
   id?: string;
 }
 
-/** @alpha A tool to define a clip volume for a view */
+/** @public A tool to define a clip volume for a view */
```

Comment:
Undocumented overrides of base class methods like `onResetButtonUp` clutter the documentation and are not really part of the API - can you mark them as internal?

Reply:
Should I just make every method @internal? I'd just like people to be able to install these tools, sub-class them to extend them isn't the goal...

Reply:
Probably. If somebody does want to subclass them, they'd better be looking at the actual code anyway, not the API docs.
I've been marking things like this as `@internal override` so I remember why they were made internal.

Reply:
done